paver.sh

```diff
@@ -552,6 +552,12 @@ detect_os() {
                 check_os_version "SUSE Linux Enterprise Server" 10 \
                     "$os_version_raw" os_version_chevah
                 OS="sles${os_version_chevah}"
+                if [ ${os_version_chevah} -eq 11 ]; then
+                    # Check for SLES 11 with OpenSSL 1.0 (aka Security Module).
+                    if [ -n $(rpm -q openssl1) ]; then
```

Comment:
if you think that PRM is ok, then we can keep it.

no need to send a warning or any message... if openssl 1.0.1 is installed, we should go with it... if is missing we should go with 0.9.8